tools/ethereum-tests/gen-inputs.js

```diff
@@ -210,6 +215,12 @@ describe('Generate inputs executor from ethereum tests GeneralStateTests\n\n', a
                         }
 
                         const currentTest = test[keysTests[y]];
+
+                        // check gas used by the tx is less than 30M
+                        if (Scalar.gt(Scalar.e(currentTest.blocks[0].blockHeader.gasUsed), zkcommonjs.Constants.BATCH_GAS_LIMIT)) {
+                            await updateNoExec(dir, newOutputName, 'tx gas > 30M', noExec);
```

Comment:
There are a few tests where the tx gas > 30M but can be generated and passed correctly (most of them is not the case).

Reply:
This parameter is the `gasUsed` by the transaction. If gas used is above 30M will behave differently in our `zkEvmDB` and in when executing them in the `zkevm-rom`

Reply:
Ohh its true 💯